### PR TITLE
Make_Chocolate.rb

### LINKED ISSUE #30: makeChocolate
We want make a package of goal kilos of chocolate. We have small bars (1 kilo each) and big bars (5 kilos each). Return the number of small bars to use, assuming we always use big bars before small bars. Return -1 if it can't be done.

```
makeChocolate(4, 1, 9) → 4
makeChocolate(4, 1, 10) → -1
makeChocolate(4, 1, 7) → 2
```

### PR DESCRIPTION
**Pull request by Anishluke92**

### CODE REVIEW

File: Make_Chocolate.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+makeChocolate(4, 1, 10) → -1
+makeChocolate(4, 1, 7) → 2
+=end
+
+def makeChocolate(smallbar, bigbar, goal)
+    totalbar = smallbar + (bigbar * 5)
+    smallbar if totalbar == goal
+        
+    totalbar - goal
+
```
Comment (danielpaul) [inline Make_Chocolate.rb:17]:
```suggestion

```

Extra space

File: Make_Chocolate.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+We want make a package of goal kilos of chocolate. We have small bars (1 kilo each) and big bars (5 kilos each). Return the number of small bars to use, assuming we always use big bars before small bars. Return -1 if it can't be done.
+
+makeChocolate(4, 1, 9) → 4
+makeChocolate(4, 1, 10) → -1
+makeChocolate(4, 1, 7) → 2
+=end
+
+def makeChocolate(smallbar, bigbar, goal)
+    totalbar = smallbar + (bigbar * 5)
+    smallbar if totalbar == goal
```
Comment (danielpaul) [inline Make_Chocolate.rb]: what is this line doing? if the condition is met, it is not doing anything

**Review by danielpaul — CHANGES_REQUESTED** [review]

File: Make_Chocolate.rb (hunk 2 above)
Comment (Anishluke92) [reply Make_Chocolate.rb]: So basically the math structure is, smallbar + (bigbar * 5)  
so when i calculate on this format i will be getting the goal kilos. then the if condition is used to check whether the totalbar ( calculated goal kilo) is equal to the target goal kilos (goal).

File: Make_Chocolate.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+makeChocolate(4, 1, 10) → -1
+makeChocolate(4, 1, 7) → 2
+=end
+
+def makeChocolate(smallbar, bigbar, goal)
+    totalbar = smallbar + (bigbar * 5)
+    if totalbar != goal    
+        totalbar - goal
+    else 
+        return smallbar
```
Comment (danielpaul) [inline Make_Chocolate.rb]: don't need return statement here because it will return the last thing it sees anyway

**Pull request merged**